packages/contracts-bedrock/test/kontrol/proofs/OptimismPortal.k.sol

```diff
@@ -50,16 +50,10 @@ contract OptimismPortalKontrol is DeploymentSummary, KontrolUtils {
         Types.OutputRootProof memory _outputRootProof =
             Types.OutputRootProof(_outputRootProof0, _outputRootProof1, _outputRootProof2, _outputRootProof3);
 
-        // After deployment, Optimism portal is enabled
-        require(optimismPortal.paused() == false, "Portal should not be paused");
-
         // Pause Optimism Portal
         vm.prank(optimismPortal.GUARDIAN());
         superchainConfig.pause("identifier");
 
-        // Portal is now paused
-        require(optimismPortal.paused(), "Portal should be paused");
-
```

Comment:
@mds1 I've removed these checks since they speed up the execution time and are exercised by the Foundry tests